### DISCUSSION TITLE
Directory glyph are not showing up correctly

### DISCUSSION THREAD

**Discussion by barslmn** [jarun/nnn#1675] [Q&A] [answered]
terminal emulator: konsole

I have installed the nerd version from the releases like this:
```
 wget -q -O- https://api.github.com/repos/jarun/nnn/releases/latest |
 	 awk '/browser_download_url/ && /nnn-nerd-static/ {gsub(/\\"/, ""); print $2}' |
 	 xargs wget -q -i- -O- |
 	 tar xzf - -O |
 	 sudo tee /usr/local/bin/nnn > /dev/null &&
 	 sudo chmod +x /usr/local/bin/nnn
```

nnn can display nerd fonts (look at the first directory in the image). However directory symbols are messed up.

![image](https://github.com/jarun/nnn/assets/43095157/2ff0b50f-ade0-49e7-a4d1-a20ca6f67147)

**Comment by N-R-K** [comment, marked as answer]
See [this issue](https://github.com/jarun/nnn/issues/1634). Nerd font v3 broke backwards compatibility with older versions. It's fixed in nnn's master branch but not released yet. So you need to use `nnn` from `master` branch if you use nerd font v3 or above.

**Reply by barslmn** [reply]
Thanks, compiling from source worked.

**Comment by N-R-K** [comment]
@jarun what do you think about cutting a "patch release" - e.g `v4.8.1` - so that distros and users can pick up the nerd font fixes without having to compile from `master`.

**Reply by jarun** [reply]
We'll have 4.9 soon. There are some fixes. Need some time.
